tests/integration/test_kontainer_drivers.py

```diff
@@ -91,7 +91,7 @@ def test_kontainer_driver_lifecycle(admin_mc, remove_resource,
             "nodeCount": 3
         })
     remove_resource(cluster)
-
+    wait_remove_resource(kd)
```

Comment:
This can't be moved here. Create a new fixture that allows you to pass a list so you can specify the delete order of things.